src/main/java/me/eccentric_nz/TARDIS/TARDISListenerRegisterer.java

```diff
@@ -318,6 +318,9 @@ TARDISInformationSystemListener registerListeners() {
             plugin.getPM().registerEvents(new BalloonListener(plugin), plugin);
             plugin.getPM().registerEvents(new InventoryHelper(plugin), plugin);
         }
+        if (plugin.getConfig().getBoolean("travel.allow_end_after_visit") || plugin.getConfig().getBoolean("travel.allow_end_after_visit")) {
```

Comment:
Oops, checking same config option - one should be `travel.allow_nether_after_visit`

Reply:
05d86b0